tests/integration/api/test_event_stream.py

```diff
@@ -491,6 +518,146 @@ def test_update_event_stream_uuid_change_allowed(
     assert response.data["uuid"] == str(new_uuid)
 
 
+@pytest.mark.django_db
+def test_create_event_stream_with_forward_events_true(
```

Comment:
the testing logic is very similar in multiple tests. Can you use pytest.mark.parametrize to consolidate them?